client/pages/applications/[id].tsx

```diff
@@ -71,9 +71,16 @@ const ApplicationDetails: NextPage = () => {
               {details.status === ApplicationStatus.WORKSHOP ? (
                 <div className='gap-2 flex'>
                   <Link href={`/applications/${id}/score-table`}>
-                    <Button variant='primary' customClass='py-2'>
+                    <a
+                      target='_blank'
+                      rel='noopener noreferrer'
+                      className={`w-auto inline-flex justify-center items-center rounded 
+  shadow-sm px-4 py-2 text-base font-bold focus:outline-none
+  disabled:opacity-50
+  focus:ring-2 focus:ring-offset-2 sm:mt-0 sm:text-sm border-transparent bg-bcBluePrimary text-white hover:bg-blue-800 focus:ring-blue-500`}
```

Comment:
Hey Karthikk, is there a particular reason why we don't just enclose a <Button> element within an <a> and get all the className info that way, so that you don't have to re-write it here again? Is it a bad practice?
<img width="334" alt="image" src="https://user-images.githubusercontent.com/102998728/217066852-4888f117-aa8d-4eac-886a-2cb82690e88f.png">

Thanks in advance!

Reply:
The issue with button is that it doesn't have href properties and the browser doesn't consider it as a link element. So because of that, had to hardcode it.

But that's a great question!